packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2760,28 +2762,27 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                             });
                     }
                 }
-            } else {
```

Comment:
Given that we had these fights in bigger Office for the long time, I think it would be great to learn from the past.
I feel like there are two ways to approach formatting:
1. We agree on specific formatting and encode it into build (linting / prettier) - having wrong format is a build break.
2. We adopt the strategy - change formatting only for code that you are touching.
   - Scope is important here. File might be that scope. But I, personally, find it difficult - I'd prefer changes to only represent "real" changes.

Thoughts?

I'd personally prefer # 1 - it's so much easier to work in Loop repo because of it - I do not even need to think about order of imports or any other stuff - tools do it for me.

+ @tylerbutler for his thoughts, as I think this topic come up many times.

Reply:
I prefer 1 as well.